stable/prometheus-pushgateway/templates/service.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ template "prometheus-pushgateway.fullname" . }}
+  annotations:
+{{ .Values.service.annotations | toYaml | indent 4 }}
```

Comment:
Please get a confirmation on `.Values.service.annotations`, in my opinion, it's should be the same with `serviceAnnotations`

Reply:
Yeah this seems to be incorrect.

Reply:
Well, Let's make this clear. Should we add the `serviceAnnotations` to the service, right?

Reply:
Thanks for catching that. I have updated the PR.